src/index.js

```diff
@@ -0,0 +1,26 @@
+import './style.css';
+import getFromLocalSTorage from './myLocalStorage.js';
+import displayTodo from './displayTodo.js';
+
+window.addEventListener('load', () => {
```

Comment:
Hi @audukehinde, this function is performing many actions. According to JavaScript clean code best practices a function should do one thing. Please change it into multiple smaller functions.

Reply:
Okay, I'll do that. Thanks